avocado/core/test.py

```diff
@@ -52,17 +52,6 @@
 COMMON_TMPDIR_NAME = 'AVOCADO_TESTS_COMMON_TMPDIR'
 
 
-class NameNotTestNameError(Exception):
-
-    """
-    The given test name is not a TestName instance
-
-    With the introduction of :class:`avocado.core.test.TestName`, it's
-    not allowed to use other types as the ``name`` parameter to a test
-    instance.  This exception is raised when this is attempted.
-    """
-
-
```

Comment:
Oh, the point here is that in Python we should not be doing type check/enforcement.  This bit was here temporarily, when the `name` parameter to `avocado.core.test.Test` got changed.  I'll add a better commit message.